packages/common/src/lib/common/password.ts

```diff
@@ -67,16 +68,16 @@ const password = pw => {
                 });
             }
         },
-        check: function (hashedPassword, callback) {
+        check: function (hashedPassword: string, callback: (err?: Error | null, isOk?: boolean) => void) {
```

Comment:
It shouldn't be necessary to repeat the function type annotations here, since they are already on the method return type.